### PR TITLE
Downgrade Gradle to 4.6 and fix broken tests.

### PR DESCRIPTION
**Pull request by mnonnenmacher**
To work around an issue with kotlintest skipping failed tests:
https://github.com/kotlintest/kotlintest/issues/336

Also fix broken tests that were not discovered due to this issue.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/575)

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
```
Comment (sschuberth) [inline gradle/wrapper/gradle-wrapper.properties]: Commit message: Should be "To work around".

File: analyzer/src/test/kotlin/PackageManagerTest.kt [hunk 2]
```diff
@@ -29,7 +29,7 @@ import io.kotlintest.specs.WordSpec
 import java.io.File
 
 class PackageManagerTest : WordSpec({
-    val projectDir = File("analyzer/src/funTest/assets/projects/synthetic/all-managers")
+    val projectDir = File("../analyzer/src/funTest/assets/projects/synthetic/all-managers")
```
Comment (sschuberth) [inline analyzer/src/test/kotlin/PackageManagerTest.kt]: Just omit "../analyzer/" from the start of the string.

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (mnonnenmacher) [reply gradle/wrapper/gradle-wrapper.properties]: Done.

File: analyzer/src/test/kotlin/PackageManagerTest.kt (hunk 2 above)
Comment (mnonnenmacher) [reply analyzer/src/test/kotlin/PackageManagerTest.kt]: Done.

**Pull request merged**